atom/src/pythonhelpers.h

```diff
@@ -24,6 +24,7 @@
 
 #define pyobject_cast( o ) ( reinterpret_cast<PyObject*>( o ) )
 #define pytype_cast( o ) ( reinterpret_cast<PyTypeObject*>( o ) )
+#define pytype_name( o ) ( PyObject_GetAttrString( pyobject_cast( o->ob_type ), "__name__" ) )
```

Comment:
The getattr call is unnecessary. You can use `o->ob_type->tp_name` directly to get the `char*` type name, and use `strcmp` to compare them.

Reply:
Addressed